evergreen/config.yml

```diff
@@ -521,6 +525,7 @@ buildvariants:
     cmake_bindir: "./cmake_binaries/CMake.app/Contents/bin"
     max_jobs: $(sysctl -n hw.logicalcpu)
     run_tests_against_baas: On
+    xcode_developer_dir: /Applications/Xcode11.3.app/
```

Comment:
```suggestion
    xcode_developer_dir: /Applications/Xcode11.3.app/Developer
```

Reply:
Is this necessary? What does this do? It looks like compile works without this.

Reply:
The [`xcode-select` manpage](https://www.manpagez.com/man/1/xcode-select/) says it's needed for historical reasons, but doesn't mention when those reasons became "historical". If it works against Xcode 11 then I guess it's safe to assume going forward we can omit it.
We specify the full path to the `Developer` folder in the `Jenkinsfile`, but that obviously grew organically over time.